org.eclipse.january.examples/src/org/eclipse/january/examples/dataset/Utils.java

```diff
@@ -42,7 +42,7 @@ public void write(int b) {
 				}
 			}));
 	
-			LoggerFactory.getLogger(BasicExample.class);
+			LoggerFactory.getLogger(Utils.class);
```

Comment:
This wasn't strictly related but was matched by the procedure I used when checking for GDA classes using other classes' loggers.

Reply:
Thanks